vscode-extension/src/util.ts

```diff
@@ -295,3 +295,19 @@ export function urlJoin(...args) {
   return normalize(parts);
 }
 //#endregion
+
+
+/**
+ * AIConfig Vscode extension has a dependency on the Python extension. 
+ * This function retrieves and returns the path to the current python interpreter.
+ * @returns the path to the current python interpreter
+ */
+export async function getPythonPath(): Promise<string> {
```

Comment:
Make this function a bit more bulletproof by putting it in a try/catch, and in the catch return `python`? Alternatively, in the `catch`, ask the user to specify the path to their python interpreter: 

```
  let config = vscode.workspace.getConfiguration(EXTENSION_NAME);
  let savedPythonPath = config.get<string>("pythonPath");

pythonPath = await vscode.window.showInputBox({
    prompt: "We couldn't determine your python installation automatically. Enter the path to your python interpreter",
    value: savedPythonPath,
    validateInput: (text) => {
      if (!text) {
        return "File path is required";
      }

      if (!isValidFilePath(text)) {
        return "File path is not valid or file does not exist";
      }

      return null;
    },
  });
```

Reply:
discussed offline, adding this ontop